package.json

```diff
@@ -84,7 +84,7 @@
     "shx": "^0.3.3",
     "snyk": "^1.653.0",
     "source-map-loader": "1.1.3",
-    "terser-webpack-plugin": "^5.1.4",
+    "terser-webpack-plugin": "^4.2.3",
```

Comment:
otherwise, get this error [https://github.com/webpack-contrib/terser-webpack-plugin/issues/335](https://github.com/webpack-contrib/terser-webpack-plugin/issues/335)